.github/workflows/ci.yml

```diff
@@ -43,8 +43,9 @@ jobs:
     - name: Install xcbeautify
       run: brew install xcbeautify
     - name: scripts/xcodebuild_wrapper.sh ${{ matrix.build_command }} | xcbeautify
-      run: scripts/xcodebuild_wrapper.sh ${{ matrix.build_command }} | xcbeautify
-      continue-on-error: true
+      run: |
+        set -o pipefail
```

Comment:
If you do "set -eox" it will also print out the script before executing. I find that helpful